build.sh

```diff
@@ -427,6 +427,13 @@ case $key in
 esac
 done
 
+
+case ${INSTALL_PREFIX} in
```

Comment:
I think we need it in Windows too.

Reply:
Yes, I will do that.

Reply:
The Windows Script currently has no input parameters. Therefore, in a later version, the input parameters should be integrated.